sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DDLParser.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.spark.sql.types._
 /**
  * A parser for foreign DDL commands.
  */
-class DDLParser(fallback: => ParserDialect)
+class DDLParser(fallback: => ParserInterface)
```

Comment:
note: it is somewhat strange that DDLParser has a "parse" method (not parsePlan). I'm leaving that unchanged because this class will likely just go away.